### PR TITLE
People list router

### PR DESCRIPTION
**Pull request by golosgit**
Added router in PeopleList for PersonDetails and additionally removed few glitches:
- fixed MovieList not displaying when there is no release date for a movie
- MovieDetails release date and production set to "N/A" if there are no results
- removed "no image" icon when there is no poster

### CODE REVIEW

File: src/features/movies/MoviesList/index.js [hunk 1]
```diff
@@ -61,11 +57,14 @@ const MoviesList = ({ insideDetails }) => {
               <StyledLink key={movie.id} to={toMovieDetails({ id: movie.id })}>
                 <TileMovie>
                   <ImageWrapper>
-                    <Image src={`https://image.tmdb.org/t/p/w400${movie.poster_path}`} alt="" />
+                    {movie.poster_path ?
+                      <Image src={`https://image.tmdb.org/t/p/w400${movie.poster_path}`} /> :
```
Comment (maxnatalia) [inline src/features/movies/MoviesList/index.js]: In my opinion we should have alt="" in Image but it is your decision according to the call.

File: src/features/people/PeopleList/index.js [hunk 2]
```diff
@@ -1,45 +1,47 @@
...
+        <Loading />
+      ) : (
+        <>
+          <Header>Popular people</Header>
+          <TilesContainer>
+            {people.map((person) => (
+              <StyledLink key={person.id} to={toPersonDetails({ id: person.id })}>
+                <TilePerson key={person.id}>
+                  <ImageWrapper>
+                    <Image src={`https://image.tmdb.org/t/p/w185${person.profile_path}`} alt="" />
```
Comment (maxnatalia) [inline src/features/people/PeopleList/index.js]: Here maybe we can also added the same ternary operator as MovieList?

**Review by maxnatalia — COMMENTED** [review]
Good job! I leave a suggestion about Image in PeopleList

File: src/features/movies/MoviesList/index.js (hunk 1 above)
Comment (golosgit) [reply src/features/movies/MoviesList/index.js]: I restored alt with "Movie poster" description and "Actor image" for people image alt.

File: src/features/people/PeopleList/index.js (hunk 2 above)
Comment (golosgit) [reply src/features/people/PeopleList/index.js]: Done. I will also add similar ternary for PersonDetail when I will add logic to the component.

**Review by maxnatalia — APPROVED** [review]
It looks ok. You can merge - good job!

**Pull request merged**